src/auth/tests.js

```diff
@@ -29,9 +29,7 @@ describe('Auth api', () => {
   });
 
   after(async () => {
-    await Promise.all([
-      User.remove({ email: 'admin@babajka.io' }),
-    ]);
+    await User.remove({ email: 'test1@babajka.io' });
```

Comment:
can you define `usersData` above and remove all users from this array, so if we'll need add more users, we can only added new user object to `usersData` and not copy-paste removing.

Reply:
done.